pull-request-checklist.md

```diff
@@ -0,0 +1,24 @@
+# Frontend Pull Request reviewer checklist
+
+- It follows the MediaMonks [frontend coding standards](https://github.com/mediamonks/frontend-coding-standards) (proper naming for variables, functions, properties, booleans, etc.)
```

Comment:
Please change the beginning of each line to:
```
- [x] 
```
then it looks like a check list

Reply:
added